yt_dlp/extractor/cda.py

```diff
@@ -83,8 +98,92 @@ def _download_age_confirm_page(self, url, video_id, *args, **kwargs):
                 'Content-Type': content_type,
             }, **kwargs)
 
+    def _login(self, username, password):
+        token = [c for c in self._downloader.cookiejar if c.domain == 'ytdlp-internal.cda.pl' and c.name == 'token']
+        valid_until = [c for c in self._downloader.cookiejar if c.domain == 'ytdlp-internal.cda.pl' and c.name == 'valid_until']
```

Comment:
Use `self._get_cookies` to get cookies.

But, I don't understand what u r trying to do here. Where does `ytdlp-internal.cda.pl` come from?

Reply:
it's set at the end of this method after getting the token, to allow persisting the obtain log-in with `--cookies`. `self._get_cookies` did not work for me